cpp/src/arrow/CMakeLists.txt

```diff
@@ -533,8 +533,8 @@ else()
   set(ARROW_BUILD_BUNDLED_DEPENDENCIES FALSE)
 endif()
 
-if(ARROW_BUILD_BUNDLED_DEPENDENCIES)
-  string(APPEND ARROW_PC_LIBS_PRIVATE " -larrow_bundled_dependencies")
+if(NOT ARROW_BUILD_BUNDLED_DEPENDENCIES)
```

Comment:
Is this back to `NOT`? I feel like it flip-flops on every commit…was there a rebase error or something? It might be easier to squash current commits if needed

Reply:
Ugh, sorry. I was switching back and forth between linux and mac to fix the mac issue and probably caused this. I'll squash later today.